src/controllers/request.js

```diff
@@ -95,4 +97,99 @@ export default class usersController {
     });
     }
   }
+  static async rejectRequest(req, res) {
+    try{
+        const { requestId } = req.query;
+        const managerId = req.body.id;
+        
+        const managerExists = await models.User.count(
+            { 
+                where: { id : managerId, role : 'manager'}
+            }
+        ).then( (count) => {
+            return (count > 0) ? true: false;
+        })
+        if(managerExists){
+            const request = await models.Request.findAll(
+                {
+                    where: {
+                        id: requestId,
+                        managerId
+                    }
+                }
+            ).then ((request) => {
```

Comment:
There is no reason for using await and then at the same time, please change that.

Reply:
Thanks @kagabof , 
I have learnt it.